### PR TITLE
Mapping slides: Update

### PR DESCRIPTION
**Pull request by gallardoalba**
This PR fixes the [position of rows](https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/mapping/slides.html#29) in a few slides and includes additional information about mapping tools.

### CODE REVIEW

File: topics/sequence-analysis/tutorials/mapping/slides.html [hunk 1]
```diff
@@ -977,6 +944,73 @@
...
+Many different tools available
+
+Different strengths and weaknesses, comparison table in link
+
+---
+class: top
+
+# Mapping tools
+
+<table style="width:100%; table-layout:fixed; font-size:0.8em;">
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/mapping/slides.html:968]: is it possible to use a markdown table here? I think we can put the css if it's needed in the slides and apply a class

File: topics/sequence-analysis/tutorials/mapping/slides.html [hunk 2]
```diff
@@ -682,7 +684,7 @@
 ]
 
 .pull-right[
-<i class="fa fa-arrows-h" style="position:absolute;font-size:3em;left:6.4em;"></i>
+<i class="fa fa-arrows-h" style="position:absolute;font-size:3em;left:8em;"></i>
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/mapping/slides.html:687]: thanks for catching these!

**Review by hexylena — APPROVED** [review]

File: topics/sequence-analysis/tutorials/mapping/slides.html (hunk 1 above)
Comment (gallardoalba) [reply topics/sequence-analysis/tutorials/mapping/slides.html:968]: Sure, I'll check it.

Comment (gallardoalba) [reply topics/sequence-analysis/tutorials/mapping/slides.html]: Done :D

**Review by hexylena — APPROVED** [review]

**Pull request merged**